keyboards/sofle/keymaps/via/keymap.c

```diff
@@ -17,9 +17,15 @@
 
 #include QMK_KEYBOARD_H
 #include "oled.c"
-#include "encoder.c"
 
-//Default keymap. This can be changed in Via. Use oled.c and encoder.c to change beavior that Via cannot change.
+#ifdef ENCODER_MAP_ENABLE
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU),           ENCODER_CCW_CW(KC_PGDN, KC_PGUP) },
```

Comment:
I believe there's a mistake in the second encoder here, it should be in this order from the original `encoder.c` code: `KC_PGUP, KC_PGDN` if `ENCODER_CCW_CW` from my understanding is from couterclockwise to clockwise @ireun

Reply:
Well, I'd say this makes more sense now, since both `KC_VOLxx` and `KC_PGxx` are in the same order. Usually in my experience rotating clockwise corresponds to `increasing`, that applies to clocks, volume knobs or even gas burners.

Reply:
What I mean is there's a change in behavior between what u had before with `encoder.c` file and the new `encoder_map`. Before, the encoder at index `1` was doing `KC_PGDN` on clockwise rotation while now it's being reversed to `KC_PGUP` on clockwise rotation.

Reply:
maybe mention this in the PR about the behavior change?

Reply:
Sure, thank You for pointing that out.